optimum/onnxruntime/trainer.py

```diff
@@ -432,11 +417,7 @@ def train(
             elif isinstance(train_dataloader.dataset, IterableDatasetShard):
                 train_dataloader.dataset.set_epoch(epoch)
 
-            if is_torch_tpu_available():
-                parallel_loader = pl.ParallelLoader(train_dataloader, [args.device]).per_device_loader(args.device)
-                epoch_iterator = parallel_loader
-            else:
-                epoch_iterator = train_dataloader
+            epoch_iterator = train_dataloader
```

Comment:
Is there a reason we introduce the variable `epoch_iterator` ? (can we just use `train_dataloader` ?)